config/checkstyle_input_suppressions.xml

```diff
@@ -0,0 +1,1063 @@
+<?xml version="1.0"?>
+
+<!DOCTYPE suppressions PUBLIC
+        "-//Checkstyle//DTD SuppressionFilter Configuration 1.1//EN"
+        "https://checkstyle.org/dtds/suppressions_1_1.dtd">
+
+<suppressions>
+
+    <!-- Suppress miscellaneous files -->
+    <suppress checks="RegexpMultiline" files="package-info.java"/>
+
+    <!-- Suppress older check UT input files -->
+    <suppress checks="RegexpMultiline" files="InputNoCodeInFile6.java"/>
```

Comment:
please do paths like `blocks[\\/]rightcurly[\\/]InputRightCurlySameNewTokens.java`
and sort paths to keep all Checks Inputs together.
It will help us to improve Inputs continuously in that backlog.